pytubefix/contrib/search.py

```diff
@@ -63,7 +67,7 @@ def __init__(
         self.po_token_verifier = po_token_verifier
 
         self._innertube_client = InnerTube(
-            client='WEB',
+            client=self.client,
```

Comment:
Some clients do not have the same pagination structure, so this will cause an error if the user changes the client.